quickswitch.py

```diff
@@ -429,6 +448,11 @@ def main():
 
     args = parser.parse_args()
 
+    global follow
+    follow = args.follow
+    global followifempty
+    followifempty = args.followifempty
```

Comment:
don't like the globals very much, but looking at the code doing this in a more functional way looks like it'll get pretty messy